sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -962,4 +962,20 @@ private void validateUserType(Request userRequest) {
               new String[] {userType, JsonKey.USER_TYPE}));
     }
   }
+
+  public void validateUserMergeRequest(Request request) {
+    if(null == request.getRequest().get("mergeeId") || StringUtils.isEmpty((String) request.getRequest().get("mergeeId"))) {
```

Comment:
1. Check StringUtils has the method for null and empty both check.
2. mergeeId : are u using fromAccountId or mergeeId

Reply:
StringUtils.isBlank serving both, used it.
In normal code i used mergee and merger, but as user parameter in request used fromAccountId and toAccountId